config/local.example.php

```diff
@@ -10,6 +10,8 @@
     'elastic_servers' => ['http://elife_search_elasticsearch:9200'],
     'annotation_cache' => false,
     'ttl' => 0,
+    'file_log_path' => __DIR__.'../var/logs/debug.log',
```

Comment:
if it's called `debug.log` it should have `debug` statements in it, not just `info`?